nextflow.config

```diff
@@ -53,6 +53,9 @@ params {
 
     // Path to temporary intermediary file used for developing genomes downloading
     filtered_hits              = "$baseDir/assets/samples.csv"
+
+    // BLAST DB path
+    blast_db                   = "/data/BLAST_DB"
```

Comment:
What do you think about `"$baseDir/data/BLAST_DB"` instead?

Reply:
I was wondering that maybe would it be even better if this is by default empty (`null`?) and the user has to provide the value either in CLI as param or fill in a profile or fill in directly? I'm wondering that since this is essential (at least currently) for running the pipeline, it would be maybe good that the user is aware of the need to set the value somewhere. What do you think?